packages/admin/src/pages/scheme/[schemeId]/[exportId]/view/[submissionId].page.tsx

```diff
@@ -0,0 +1,125 @@
+import { GetServerSidePropsContext } from 'next';
+import { getLoggedInUsersDetails } from '../../../../../services/UserService';
+import InferProps from '../../../../../types/InferProps';
+import { generateErrorPageRedirect } from '../../../../../utils/serviceErrorHelpers';
+import { getSessionIdFromCookies } from '../../../../../utils/session';
+import CustomLink from '../../../../../components/custom-link/CustomLink';
+import Meta from '../../../../../components/layout/Meta';
+import { QuestionRow } from './QuestionRow.component';
+import { SectionData } from '../../../../../types/SubmissionSummary';
+import { getFailedExportDetails } from '../../../../../services/ExportService';
+
+export const getServerSideProps = async ({
+  req,
+  res,
+  query,
+}: GetServerSidePropsContext) => {
+  const { schemeId, exportId, submissionId } = query as Record<string, string>;
+
+  const errorPageRedirect = generateErrorPageRedirect(
+    'Something went wrong.',
+    req.headers.referer ? req.headers.referer : '/dashboard',
+    !!req.headers.referer
+  );
+
+  let userDetails, submission, sessionCookie;
+
+  try {
+    sessionCookie = await getSessionIdFromCookies(req);
+    userDetails = await getLoggedInUsersDetails(sessionCookie);
```

Comment:
This additional call to the backend can be removed and you can get the organisation’s name from the submission object (`submission.legalName`) returned from `getFailedExportDetails`